### PR TITLE
Use radio buttons on equality and diversity start page

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

The equality and diversity start page has seen many revisions, but none have addressed us using a button and a link to direct users to different routes – ~~a situation that’s been made worse since the nearby inclusion of a link to our privacy policy as well~~ or would had that link appeared 😱.

## Changes proposed in this pull request

* Use radio buttons to show the two choices instead of a link and a button
* Fix link to privacy policy not being rendered
* Update tests to reflect new behaviour on this page
* Localise most of the strings in this section (which also ensures section title is used consistently across pages in this section)
* Use new `caption` attribute in the form builder, removing need for `equality_and_diversity_caption` helper
* Move grid layout out from review component and into view

## Guidance to review

Perhaps best to view each commit individually - the one that localises strings obviously touches a lot of code.

| Before | After |
| - | - |
| e q-old | e q-new |

## Link to Trello card

https://trello.com/c/jenqB5nt

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
Hmmm, wonder if there should be a test to ensure privacy policy _is_ linked to…

**Comment by paulrobertlloyd** [comment]
> Hmmm, wonder if there should be a test to ensure privacy policy _is_ linked to…

Added in https://github.com/DFE-Digital/apply-for-teacher-training/pull/3242/commits/e52ba8e35589f0b47dcbcc3158acfb4cdac1a6c4

**Review by stevehook — APPROVED** [review]
:+1: Review app looks good. Some minor suggestions

File: spec/support/test_helpers/candidate_helper.rb [hunk 1]
```diff
@@ -88,7 +88,8 @@ def candidate_submits_application
 
     click_link 'Check and submit your application'
     click_link 'Continue'
-    click_link 'Continue without completing questionnaire'
+    find(:css, '#candidate-interface-equality-and-diversity-choice-form-choice-no-field').set(true)
```
[the remark this answers is not in the document]
Comment (paulrobertlloyd) [reply spec/support/test_helpers/candidate_helper.rb:91]: 😂

File: app/forms/candidate_interface/equality_and_diversity/disabilities_form.rb [hunk 2]
```diff
@@ -3,13 +3,13 @@ class EqualityAndDiversity::DisabilitiesForm
...
-      ['mental', 'Mental health condition'],
-      ['physical', 'Physical disability or mobility issue'],
-      ['social', 'Social or communication impairment'],
+      ['blind', I18n.t('equality_and_diversity.disabilities.blind.label')],
+      ['deaf', I18n.t('equality_and_diversity.disabilities.deaf.label')],
+      ['learning', I18n.t('equality_and_diversity.disabilities.learning.label')],
+      ['long_standing', I18n.t('equality_and_diversity.disabilities.long_standing.label')],
+      ['mental', I18n.t('equality_and_diversity.disabilities.mental.label')],
+      ['physical', I18n.t('equality_and_diversity.disabilities.physical.label')],
+      ['social', I18n.t('equality_and_diversity.disabilities.social.label')],
```
[the remark this answers is not in the document]
Comment (paulrobertlloyd) [reply app/forms/candidate_interface/equality_and_diversity/disabilities_form.rb:12]: I like it!

**Pull request merged**